java-frontend/src/test/java/org/sonar/java/SonarComponentsTest.java

```diff
@@ -924,20 +925,18 @@ private static Stream<Arguments> fileCanBeSkipped_only_logs_on_first_call_input(
   @ParameterizedTest
   @MethodSource("fileCanBeSkipped_only_logs_on_first_call_input")
   void fileCanBeSkipped_only_logs_on_the_first_call(SonarComponents sonarComponents, InputFile inputFile, String logMessage) throws IOException {
-    // logs may be empty or contain some progress log line
-    assertThat(
-      filterOutAnalysisProgressLogMessages(logTester.getLogs(Level.INFO))).isEmpty();
+    assertThat(logTester.logs(Level.INFO)).isEmpty();
 
     SensorContext contextMock = mock(SensorContext.class);
     sonarComponents.setSensorContext(contextMock);
     when(inputFile.contents()).thenReturn("");
     sonarComponents.fileCanBeSkipped(inputFile);
-    List<String> logs = filterOutAnalysisProgressLogMessages(logTester.getLogs(Level.INFO));
+    List<String> logs = logTester.rawMessages(Level.INFO);
     assertThat(logs).hasSize(1);
     assertThat(logs.get(0)).isEqualTo(logMessage);
 
     sonarComponents.fileCanBeSkipped(inputFile);
-    logs = filterOutAnalysisProgressLogMessages(logTester.getLogs(Level.INFO));
```

Comment:
`org.sonar.java.TestUtils#filterOutAnalysisProgressLogMessages` seems to not be used anymore, we should delete the method definition.